pkgs/applications/networking/browsers/lagrange/default.nix

```diff
@@ -15,23 +17,28 @@
 
 stdenv.mkDerivation rec {
   pname = "lagrange";
-  version = "1.5.2";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
     owner = "skyjake";
     repo = "lagrange";
     rev = "v${version}";
-    sha256 = "sha256-NjiTjY2YuxUs/Wny7aDqHGw/2ML1fenjHrl089rLXFI=";
+    sha256 = "sha256-WBG9xmkBdycKR6Eg6ljH0/JK2jxL+37oycvkM3L34wU=";
     fetchSubmodules = true;
   };
```

Comment:
```suggestion
  };

  postPatch = ''
    rm -r lib/fribidi lib/harfbuzz
  '';
```

Reply:
Why? Build process will not use it anyway, since `-DENABLE_HARFBUZZ_MINIMAL:BOOL=OFF` and `-DENABLE_FRIBIDI_BUILD:BOOL=OFF`.

Reply:
Imagine he's just suggesting a belt-and-braces approach.

Reply:
To make sure it can't use it by accident or mistakes in the build script.

Reply:
Point taken.